src/Athena.Importer/data/course/mthm270.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id": "ca874843-d2dc-404d-aab1-f284032417e9",
+  "name": "MTHM  270 - DISCRETE MATHEMATICS",
```

Comment:
Strip extra space from name (Between `MTHM` and `270`)

Reply:
DONE

Reply:
Lies, it's still in the diff :)